mobile-save-for-later/conf/cfn.yaml

```diff
@@ -1,16 +1,4 @@
 Resources:
-  ApiRoute53:
```

Comment:
We now create this in TypeScript via `CfnRecordSetGroup`

Reply:
As we import the resources from `cfn.yaml` and produce a single template AWS is unaware of this change (importantly the [logical id is unchanged](https://github.com/guardian/cdk/blob/main/docs/stateful-resources.md#changes-to-a-resources-logical-id)).